lib/ses-transport.js

```diff
@@ -0,0 +1,54 @@
+// custom Winston transport for AWS SES (Simple Email Service) using the AWS JavaScript SDK
+// based a now-outdated outdated package https://github.com/jpgarcia/winston-amazon-ses
+
+const Transport = require('winston-transport');
+const nodemailer = require('nodemailer')
+const aws = require('aws-sdk')
+const util = require('util')
+
+module.exports = class SESTransport extends Transport {
+  constructor(opts) {
+    super(opts);
+    this.opts = opts
+
+    if (this.opts.silent) {
+      callback()
+      return
+    }
+
+    aws.config.update({
+      accessKeyId: opts.accessKeyId,
+      secretAccessKey: opts.secretAccessKey,
+      region: opts.region,
+    })
+
+    var transport = {
+      SES: new aws.SES({
+        apiVersion: '2010-12-01'
```

Comment:
Not sure why this version name has 2010; same as in mailer.js

Reply:
Even according to the official AWS SDK, we should still be using the 2010 SDK:

https://docs.aws.amazon.com/ses/latest/APIReference/Welcome.html